services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -145,6 +145,10 @@ async def dispatch_update_for_directory(directory_path: Path) -> None:
 # INPUTS section
 
 
+def _is_zip_file(file_path: Path) -> bool:
+    return str(magic.from_file(file_path, mime=True)) == "application/zip"
```

Comment:
```suggestion
    return f"{magic.from_file(file_path, mime=True)}" == "application/zip"
```

Reply:
I will put it on 2 lines and use f-strings